From ebb9336926af0c5becc49b64c7b15e8102565b67 Mon Sep 17 00:00:00 2001
From: Hao Zheng <haoz@marvell.com>
Date: Wed, 4 Sep 2019 15:03:47 -0700
Subject: [PATCH 340/767] octeontx2-af: kpu profile update for protocol nvgre

commit e849a34c43680a5dbe0bf18bc1317e0d290de3e2 from
git@git.assembla.com:cavium/WindRiver.linux.git

update kpu profile so that nvgre has a separate ltype LD_NVGRE
instead of combined with LD_GRE

Change-Id: I3ef48896470cfed83c203e430915252c359db210
Signed-off-by: Hao Zheng <haoz@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/15508
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Tested-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/npc_profile.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/npc_profile.h b/drivers/net/ethernet/marvell/octeontx2/af/npc_profile.h
index 852a7fc9ff22..9cb7f68b5b9f 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/npc_profile.h
+++ b/drivers/net/ethernet/marvell/octeontx2/af/npc_profile.h
@@ -10711,7 +10711,7 @@ static struct npc_kpu_profile_action kpu8_action_entries[] = {
 		NPC_ERRLEV_RE, NPC_EC_NOERR,
 		12, 16, 20, 2, 0,
 		NPC_S_KPU11_TU_ETHER, 8, 1,
-		NPC_LID_LD, NPC_LT_LD_GRE,
+		NPC_LID_LD, NPC_LT_LD_NVGRE,
 		NPC_F_LD_L_GRE_NVGRE,
 		0, 0, 0, 0,
 	},
-- 
2.31.1

